docs/blog/_posts/2018-07-06-ninth-dotty-milestone-release.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.9.0-RC1
+author: Allan Renucci
+authorImg: /images/allan.jpg
+date: 2018-07-06
+---
+
+Today, we are excited to release Dotty version 0.9.0-RC1. This release serves as a technology
+preview that demonstrates new language features and the compiler supporting them.
+
+Dotty is the project name for technologies that are considered for inclusion in Scala 3. Scala has
+pioneered the fusion of object-oriented and functional programming in a typed setting. Scala 3 will
+be a big step towards realizing the full potential of these ideas. Its main objectives are to
+
+- become more opinionated by promoting programming idioms we found to work well,
```

Comment:
Not opinionated. 

Its main objectives are the following:
  
- incorporate modern programming idioms that satisfy current and future needs,
- ...

Reply:
Ok